engine/src/flutter/docs/Crashes.md

```diff
@@ -1,16 +1,16 @@
-## Symbolicating stack traces for engine crashes
+# Symbolicating stack traces for engine crashes
 
 iOS app archives produced using Flutter 3.24 or later embed engine debugging symbols and thus crashes are symbolicated by default.
 
 The easiest way to symbolicate stack traces for Android and older iOS apps is to run the [dart_ci symbolizer locally](https://github.com/dart-lang/dart_ci/blob/main/github-label-notifier/symbolizer/README.md#using-this-locally). If that is not an option, the steps below explain how to do it manually.
 
-### Android
+## Android
 
-#### Get the symbols
+### Get the symbols
 
 1. Get the Flutter Framework or Flutter Engine revision from the report. If you have the Engine revision, skip to step 3.
 
-2. Get the Engine revision from the Framework (this could be automated). https://github.com/flutter/flutter/blob/main/bin/internal/engine.version is the file which contains the information. Substitute the framework hash for `main` in that url.
+2. Get the Engine revision from the Framework (this could be automated). `flutter/blob/main/bin/cache/engine.stamp` is the file which contains the information. Substitute the framework hash for `main` in that url.
```

Comment:
engine.stamp is ephemeral - its created by scripts and not checked in

Reply:
this was a dumb comment - its `bin/cache/engine-dart-sdk.stamp` that I was thinking of.